__tests__/app.test.js

```diff
@@ -40,12 +40,14 @@ describe("GET api/topics", () => {
         body.forEach((item) => {
           return expect(item.hasOwnProperty("description")).toBe(true);
         });
-        body.forEach(item => {
-           expect(item).toEqual(expect.objectContaining({
-                slug: expect.anything(),
-                description: expect.anything()
-            }))
-        })
+        body.forEach((item) => {
```

Comment:
I'm going to say the same here as the previous PR, if you're using a forEach, you must also have an expect asserting a length of at least one :)